acnportal/acnsim/base.py

```diff
@@ -14,7 +14,7 @@
 import warnings
 import pkg_resources
 import pandas
-from pandas.io.common import stringify_path, get_handle, get_filepath_or_buffer
+from pandas.io.common import stringify_path, get_handle, _get_filepath_or_buffer
```

Comment:
I'm a bit concerned about backward compatibility here. Users running pandas 1 will get an error here. Would it be possible to keep this working for pandas 1 by checking pandas install version but throw a deprecation warning if the user is using pandas 1?